workshop/code_scan/README.md

```diff
@@ -32,7 +34,14 @@ Malicious actors can exploit vulnerabilities in your code to gain unauthorized a
 
 ## Tools Used in This Module
 
-- [**Dependency Check**](https://github.com/dependency-check/DependencyCheck) - OWASP tool for Software Composition Analysis (SCA) to identify known vulnerabilities in dependencies
+- [**CodeQL**](https://github.com/github/codeql){:target="_blank"} - GitHub's semantic code analysis engine for deep security analysis
```

Comment:
It looks like the target blank does not work in Github's markdown:

<img width="748" height="162" alt="Image" src="https://github.com/user-attachments/assets/6ad34247-d9f9-441e-9831-016c35fa010e" />